### ISSUE TITLE
To begin the interactive learning mode from the normal interaction in the command line chat

### ISSUE THREAD

**Issue by Danijo** [RasaHQ/rasa_core#1222]
**Rasa Core version**:
0.9.6
**Python version**: 
3.5
**Operating system** (windows, osx, ...):
windows 7
**Issue**:
Hello,

So my issue is i would like to begin the interactive learning mode from the normal interaction in the command line chat. For this, i added a simple function in the train_online file that does the same as the main. This test didn't work and i would wondering if there is a simple fix or will i have more work to do in order for this to work (eventually, maybe can't be done).

Daniel.

**Content of domain file** (if used & relevant):
```yaml

```

**Comment by akelad** [comment]
I'm not sure what you mean, but you have to start interactive mode with the `--interactive` flag. Looks like you're using a very old version of Core though, I'd suggest upgrading and then you can take a look at the docs here: https://rasa.com/docs/core/interactive_learning/

**Comment by tmbo** [comment]
This is more a suggestion than an issue, @akelad :wink:

I think it can be done, at least I don't see a reason at the moment why this won't work. Both interfaces (cmdline, interactive learning) start a rasa core server and communicate with that through the HTTP API - so yes we could pass an existing tracker (your conversation) to the interactive learning and then continue in there. How would the interaction afterwards look like, would you go back to normal chatting?

**Comment by akelad** [comment]
my bad, i misread that 😄

**Comment by VaimeaLoja** [comment]
Yes, the idea is for the cmdline, in a normal conversation, be used also as a learning interface. Possibly as result of a "normal" conversation between the chatbot and the user. (Of course, same adjustments should be made in order for this to happen).
